components/eamxx/src/share/io/tests/io_basic.cpp

```diff
@@ -247,7 +248,7 @@ void read (const std::string& avg_type, const std::string& freq_units,
   Real fill_out;
   for (const auto& fn: fnames) {
     scorpio::get_variable_metadata(filename,fn,"_FillValue",fill_out);
-    REQUIRE(fill_out==DEFAULT_FILL_VALUE);
+    REQUIRE(fill_out==constants::DefaultFillValue<float>().value);
```

Comment:
This is fine, but it is customary to access compile-time constants as such:
```
   REQUIRE (fill_out==constants::DefaultFillValue<float>::value);
```

Reply:
I tried the `::value` route first but kept running into compiler time errors.  So I settled on this since it worked.  Maybe I am doing something wrong in my definition of `DefaultFillValue`?